src/main/java/bot/telegram/state/BotMessageSendHinter.java

```diff
@@ -3,9 +3,9 @@
 public enum BotMessageSendHinter {
```

Comment:
`-er` postfix should not be utilized here. `-er` postfix is applicable to business services, this enum is, in its essense, the model, there is not logic out there. Consider removing this postfix.